apps/server/src/modules/tldraw/controller/tldraw.ws.ts

```diff
@@ -45,4 +65,13 @@ export class TldrawWs implements OnGatewayInit, OnGatewayConnection {
 		const urlStripped = request.url.replace(/(\/)|(tldraw-server)/g, '');
 		return urlStripped;
 	}
+
+	public parseCookiesFromHeader(request: Request): { [p: string]: string } {
```

Comment:
there seems to be no reason why this method is public. it should be private

Reply:
this method is public for testing purposes

Reply:
api tests are also a good way to ensure coverage